### ISSUE TITLE
Support non-sortable elements inside ui-sortable container

### ISSUE THREAD

**Issue by huston007** [angular-ui/ui-sortable#434]
It is a really unuseful limitation to have only sortable items inside container
```
ui-sortable element should only contain one ng-repeat and not any other elements (above or below).
Otherwise the index matching of the generated DOM elements and the ng-model's items will break.
In other words: The items of ng-model must match the indexes of the generated DOM elements.
```

And it is easy to fix I believe

**Comment by thgreasi** [comment]
First of all, thanks for your effort on this issue and your PR. I think
that this has been addressed to some point in the v0.14 branch. (It's 100%
my fault that v0.14 hasn't been merged yet.) Can you check that it works
for you? If you think that it's ok then we could merge it to master in the
weekend.

I would appreciate any feedback on the v0.14 branch implementation or even
any PR that might improve it feature structure wise (just ensure that it
targeted against the v0.14 branch).

Once again thanks for your contribution to this library.

> --
*Thodoris Greasidis*
Computer, Networking &
Communications Engineer

**Comment by huston007** [comment]
@thgreasi Yeah it looks like 0.14 have the same way of items matching. I'll try it soon, thank you!
